runelite-client/src/main/java/net/runelite/client/plugins/playerindicators/PlayerIndicatorsPlugin.java

```diff
@@ -249,6 +254,64 @@ private String decorateTarget(String oldTarget, Decorations decorations)
 		return newTarget;
 	}
 
+	/*
+		Changes the player name for the trade window title to be green if they are a friend.
+	 */
+
```

Comment:
I don't think this comment is very helpful here. If kept, it should be moved inside of the event subscriber block.